docs/integration/solr.md

```diff
@@ -1,8 +1,114 @@
 title: Solr Monitoring Integration
 description:  Our monitoring and logging platform includes integration for Solr. Use predefined key metrics reports combined with rich data visualization tools to monitor critical Solr issues, and receive alerts on memory usage, uptime, load averages, index stats, document and filter caches, latency, rate, and more
 
+Sematext offers simple and versatile Solr monitoring agent written in Java and Golang without CPU and memory overhead on your application. It's easy to install and require no changes in Solr source code or your application source code. 
+
+## Sematext Solr Monitoring Agent 
+This lightweight, open-source [Java Monitoring Agent](https://github.com/sematext/sematext-agent-java) collects Kafka performance metrics and sends them to Sematext. It comes packaged with a Golang-based agent responsible for Operating System level metrics like network, I/O and more. The Solr Monitoring Agent can be installed with RPM/DEB package manager on any host running Linux or in a containerized environment using ```sematext/spm-client```.
```

Comment:
Kafka?  Or Solr?
I/O => disk I/O
and more => , and more

Reply:
Adjusted.